### ISSUE TITLE
pb avec ajout a posteriori d'une règle n'affichant pas des étiquettes déplacées

### ISSUE THREAD

**Issue by FERRATON** [Oslandia/labelconnector#29]
si on ajoute, après avoir déplacé des étiquettes, une règle qui n'affiche plus certaines étiquettes déplacées, le connecteur reste visible. Je pense que ce pb ne doit pas être facile à résoudre.
Au minimum avertir les utilisateurs dans la documentation de déplacer les étiquettes avant ajout de toute règles n'affichant pas toutes les étiquettes.
![2019-07-18_16h03_09](https://user-images.githubusercontent.com/2129643/61464050-e46b0f00-a975-11e9-8d36-0b629ce50e9a.png)

**Comment by FERRATON** [comment]
euh... le contraire... écrire les règles avant de déplacer les étiquettes !

**Comment by haubourg** [comment]
Oui, je ne vois pas de solution simple, hormis de reproduire les règles de visibilité des étiquettes sur la classe de symbole manuellement.
